components/ScheduleTable.tsx

```diff
@@ -0,0 +1,310 @@
+import React, { useMemo } from 'react';
+// import Link from 'next/link';
+import { useTable, usePagination } from 'react-table';
+import styled from 'styled-components';
+import { Game } from '../pages/api/v1/schedule';
+import { Team } from '..';
+import Skeleton, { SkeletonTheme } from 'react-loading-skeleton';
+import { BsChevronLeft, BsChevronRight } from 'react-icons/bs';
+
+interface Columns {
+  Header: string;
+  id?: string;
+  title?: string;
+  accessor: string | ((gameData) => string) | ((gameData) => JSX.Element);
+}
+
+interface ColumnData {
+  Header: string;
+  id?: string;
+  columns: Array<Columns>;
+}
+
+interface Props {
+  games: Array<Game>;
+  teamlist: Array<Team>;
+  isLoading: boolean;
+}
+
+function ScheduleTable({
```

Comment:
I personally think we could do a table, but it may make more sense to have it be split by in-game days. Within this table there are no mentions of dates and such but these are provided in the data. We could do something like get the first "date" of games and then for each game subtract from that and add one to get the day of the season in-game. So like if the season starts 10-12-2020 and we are on day 30-12-2020 we would be on "day 21" within the season. And users would be able to page by like every 3 days or something. starting with the most recently simmed days. However that can be a future goal. 

As for right now can we maybe add a column for "date" or find a way to do "in-game date"

Reply:
There are no dates in the data we get back from the schedule index API endpoint. But I can see now there's one in the header endpoint. I'll update the query in the index endpoint to include the date.

I like the idea of having separate tables for each sim day. But I'm wondering how much work that'd require. Each season start we'd need to request the full schedule from each simmer. We need their schedule to know exactly which games will be simmed on which IRL day so we can display them on the right day on this page. We could much easier have separate tables for each in-game day to make it look more official. But it wouldn't be as helpful to users.

Reply:
I think having tables for this is kinda unnecessary since it is very basic data. We can probably get by with a component that displays team names, winner, team logos on the left side and right side with the score. Something like this on the nhl website would look great. If you want to just put the data there, I can work on designing the look of the component.

<img width="440" alt="Screen Shot 2020-12-06 at 2 12 08 PM" src="https://user-images.githubusercontent.com/24704789/101289888-301d9300-37cd-11eb-9978-481923160e67.png">